graylog2-server/src/test/java/org/graylog2/indexer/fieldtypes/IndexFieldTypesListServiceTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.indexer.fieldtypes;
+
+import com.google.common.collect.ImmutableSet;
+import org.graylog2.indexer.MongoIndexSet;
+import org.graylog2.indexer.fieldtypes.utils.FieldTypeDTOsMerger;
+import org.graylog2.indexer.indexset.CustomFieldMapping;
+import org.graylog2.indexer.indexset.CustomFieldMappings;
+import org.graylog2.indexer.indexset.IndexSetConfig;
+import org.graylog2.indexer.indexset.IndexSetService;
+import org.graylog2.rest.models.tools.responses.PageListResponse;
+import org.graylog2.rest.resources.entities.Sorting;
+import org.graylog2.rest.resources.system.indexer.responses.IndexSetFieldType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyNoInteractions;
+
+@ExtendWith(MockitoExtension.class)
+class IndexFieldTypesListServiceTest {
+
+    IndexFieldTypesListService toTest;
+
+    @Mock
+    IndexFieldTypesService indexFieldTypesService;
+    @Mock
+    IndexSetService indexSetService;
+    @Mock
+    MongoIndexSet.Factory indexSetFactory;
+    @Mock
+    FieldTypeDTOsMerger fieldTypeDTOsMerger;
```

Comment:
I wouldn't mock this field here if it's not really used and usage of the proper class doesn't cause any issue.